mobile/src/src/components/Create/CreateCommunity/CommunityContainer.jsx

```diff
@@ -1,19 +1,493 @@
 import React from 'react';
-import { StyleSheet, View, Text } from 'react-native';
+import {
+  StyleSheet,
+  View,
+  Text,
+  ScrollView,
+  TextInput,
+  Platform,
+} from 'react-native';
+import Header from '../../common/Header';
+import PostTypeOptionsButton from './PostTypeOptionsButton';
+import BasicCreateForm from '../common/BasicCreateForm';
+import {
+  EventsEndpoint,
+  PollOptionsEndpoint,
+  PollsEndpoint,
+  PostsEndpoint,
+  ProfileEndpoint,
+} from '../../../utils/endpoints';
+import { Card, IconButton, Button, Portal, Modal } from 'react-native-paper';
+import DateTimePicker from '@react-native-community/datetimepicker';
 
-const CommunityContainer = () => {
-  return (
-    <View styles={styles}>
-      <Text>Create a post in community!</Text>
-    </View>
-  );
-};
+import CreatePageTextInput from '../common/CreatePageTextInput';
+import ButtonWithDownArrow from '../common/ButtonWithDownArrow';
+
+class CommunityContainer extends React.Component {
+  state = {
+    user: { profile_photo: null },
+    selectedPostType: 'Post',
+    setSelectedPostType: (newValue) => {
+      this.setState({ selectedPostType: newValue });
+    },
+    community: null,
+    title: '',
+    body: '',
+    pollOptions: [],
+    eventTime: new Date(),
+    eventLocation: '',
+    eventAdmissionPrice: '',
+    showDatePicker: false,
+    showTimePicker: false,
+  };
+
+  resetPage = () => {
+    this.setState({
+      title: '',
+      body: '',
+      pollOptions: [],
+      eventTime: new Date(),
+      eventLocation: '',
+      eventAdmissionPrice: '',
+      showDatePicker: false,
+    });
+  };
+
+  componentDidMount = async () => {
+    await this.getProfile();
+    this.setState({ community: this.state.user.community });
+  };
+
+  getProfile = async () => {
+    await ProfileEndpoint.get()
```

Comment:
I'm not too sure, but I feel like you could probably use the getUserData() function instead of ProfileEndpoint.get()?